src/main/java/TextView.java

```diff
@@ -34,6 +38,13 @@ public void update(Observable o, Object arg)
 	 */
```

Comment:
- [ ] Specify in the comment, that you are also printing other game info (who's turn, number of turns, etc...)